### DISCUSSION TITLE
Base URL documentation macro

### DISCUSSION THREAD

**Discussion by ibnishak** [TiddlyWiki/TiddlyWiki5#6321] [General]
Currently all links to official plugin demos are broken in offline version. See  #6319

Suggested solution: A baseurl documentation macro referring to whether it is a pre-release or stable version. The macro can be concatenated with rest to url to derive correct links. 

I am not familiar with the build script used by TW5. I am sure there are ways to correct the value of macro automatically on version bumps.

**Comment by Jermolene** [comment]
Hi @ibnishak I was thinking the macro would check for being on a file:// URI, and if so use the full URL, otherwise just use the relative one.
